forge-game/src/main/java/forge/game/GameActionUtil.java

```diff
@@ -853,12 +857,28 @@ public static CardCollectionView orderCardsByTheirOwners(Game game, CardCollecti
                     subList.add(c);
                 }
             }
+            if (sa != null && sa.getActivatingPlayer() == p && sa.hasParam("StaticEffect")) {
+                // create helper card for ordering
+                eff = new DetachedCardEffect(sa.getHostCard(), "Static Effect of " + sa.getHostCard());
+                subList.add(eff);
```

Comment:
This Detached Card Effect is not the same Effect used for GameAction, right?

This one is just for filtering?

Reply:
indeed
in theory it might be reused but that would require more logic than worth it